canonical/ubuntu.com
# [WD-7064] Chiselled ubuntu

## Done

- Landing page for chiselled Ubuntu

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Navigate to /containers/chiselled
 
## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7064

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
